tests/unit/type/list.tcl

```diff
@@ -60,8 +60,55 @@ start_server [list overrides [list save ""] ] {
         assert_equal [string repeat d 500] [r lindex list3 0]
     }
 
+    r debug quicklist-packed-threshold 5b
+
+    test {plain check LSET with combinations} {
```

Comment:
maybe add a few comments as to what exactly you aim to test, either here at the top, or next to each block.

Reply:
@oranagra , do you mean for each and every test ? or in general ?